cypress/integration/data-import/match-on-instance-identifier-scen1.spec.js

```diff
@@ -0,0 +1,143 @@
+import MatchProfiles from '../../support/fragments/data_import/match_profiles/matchProfiles';
+import FieldMappingProfiles from '../../support/fragments/data_import/mapping_profiles/fieldMappingProfiles';
+import NewActionProfile from '../../support/fragments/data_import/action_profiles/newActionProfile';
+import ActionProfiles from '../../support/fragments/data_import/action_profiles/actionProfiles';
+import NewJobProfile from '../../support/fragments/data_import/job_profiles/newJobProfile';
+import InstanceRecordView from '../../support/fragments/inventory/instanceRecordView';
+import InventorySearch from '../../support/fragments/inventory/inventorySearch';
+import permissions from '../../support/dictionary/permissions';
+import Users from '../../support/fragments/users/users';
+
+describe('ui-data-import: Match on Instance identifier match meets both the Identifier type and Data requirements (Scenario 1)', () => {
+  let userId;
+  const filePathForCreateInstance = 'ID Match Test File - Create.mrc';
+  const filePathForUpdateInstance = 'ID Match Test File - Update1.mrc';
+  const fileNameForCreateInstance = `C347828autotestFile.${getRandomPostfix()}.mrc`;
+  const fileNameForUpdateInstance = `C347828autotestFile.${getRandomPostfix()}.mrc`;
+  const jobProfileToRun = 'Default - Create instance and SRS MARC Bib';
+  const matchProfileName = `autotestMatchProf${getRandomPostfix()}`;
+  const mappingProfileName = `autotestMappingProf${getRandomPostfix()}`;
+  const actionProfileName = `autotestActionProf${getRandomPostfix()}`;
+  const jobProfileName = `autotestJobProf${getRandomPostfix()}`;
+  const instanceGeneralNote = 'IDENTIFIER UPDATE 1';
+  const resourceIdentifiers = [
+    { type: 'UPC', value: 'ORD32671387-4' },
+    { type: 'OCLC', value: '(OCoLC)84714376518561876438' },
+    { type: 'Invalid UPC', value: 'ORD32671387-4' },
+    { type: 'System control number', value: '(AMB)84714376518561876438' },
+  ];
+  const matchProfile = {
+    profileName: matchProfileName,
+    incomingRecordFields: {
+      field: '024',
+      in1: '1',
+      in2: '*',
+      subfield: 'a'
+    },
+    matchCriterion: 'Exactly matches',
+    existingRecordType: 'INSTANCE',
+    instanceOption: 'Identifier: UPC',
+  };
+  const mappingProfile = {
+    name: mappingProfileName,
+    incomingRecordType: 'MARC Bibliographic',
+    folioRecordType: 'Instance',
+    discoverySuppress: 'Mark for all affected records',
+    catalogedDate: '"2021-12-01"',
+    catalogedDateUI: '2021-12-01',
+    instanceStatus: '"Batch Loaded"',
+    instanceStatusUI: 'Batch Loaded',
+  };
+  const actionProfile = {
+    name: actionProfileName,
+    typeValue : NewActionProfile.folioRecordTypeValue.instance,
+    action: 'Update (all record types except Orders, Invoices, or MARC Holdings)'
+  };
+  const jobProfile = {
+    profileName: jobProfileName,
+    acceptedType: NewJobProfile.acceptedDataType.marc
+  };
+
+  before(() => {
+    cy.createTempUser([
```

Comment:
Here you are using cy.getAdminToken function. There is no need to call this function in the line 85. Please move the block of code from 87 line to 82 line.
Any time you call login function(cy.getAdminToken, cy.login) you reset the previous login session.

Reply:
Actually by calling on line 85  the function cy.getAdminToken(); you reset the session of login the user from the line 83